README.md

```diff
@@ -445,6 +445,11 @@ services:
         after:
             - <other service name>
 
+        # (Optional) The amount of time afforded to this service to handle
+        # SIGTERM and exit gracefully before being force killed.
+        # Default is 25 seconds ("25s").
+        kill-delay: <grace-period>
+
```

Comment:
It's weird to place this between "after" and "before". I would probably put it last, after "backoff-limit".

Also, can we clarify that this is connected to stop/restart? Perhaps:

```
        # (Optional) When a service is being stopped or restarted, this is the
        # amount of time allowed for the service to handle SIGTERM and exit
        # gracefully before being force-killed with SIGKILL. Default is 25
        # seconds ("25s").
```

Reply:
Oh, one more thing: Instead of saying `<grace-period>` as the value we should say `<duration>` (like other duration fields).